redis_session_manager.py

```
@@ -0,0 +1,127 @@
+import redis
+import os
+import json
+from typing import Dict, Any, Optional
+import time
+from dotenv import load_dotenv
+
+load_dotenv()
+REDIS_HOST =os.getenv("Redis_Host")
```

Ruff (0.8.2)

Use capitalized environment variable `REDIS_HOST` instead of `Redis_Host`

(SIM112)